snipeit.sh

```diff
@@ -492,18 +395,11 @@ case $distro in
 		sudo chmod -R 755 $webdir/$name/public/uploads
 		sudo chown -R apache:apache $webdir/$name
 
-#TODO detect if SELinux is enabled to decide what to do.
+		#TODO detect if SELinux and firewall are enabled to decide what to do
+		#Add SELinux and firewall exception/rules. Youll have to allow 443 if you want ssl connectivity.
 		# chcon -R -h -t httpd_sys_script_rw_t $webdir/$name/
-
-    #Check if iptables is running
-    /sbin/service iptables status >/dev/null 2>&1
-    if [ $? = 0 ]; then
```

Comment:
I added this in #2674. We check if iptables is running(it does by default in 6.8 minimal) and http/https ports.